neuro_flow/context.py

```diff
@@ -1306,13 +1403,20 @@ async def _get_job(
             assert isinstance(tmp_port_forward, list)
             port_forward = tmp_port_forward
 
+        image = await job.image.eval(ctx)
```

Comment:
Would be nice to have an additional check for the _potential_ image absence earlier, at the `RunningBatchFlow.create()` call time.
Like you we already have for early task checker.

I'm thinking about a new CLI command for quick statically check for YAML file.

Reply:
This is live mode code, as it was completely evaluated during `neuro run`, I think it's ok to check this here.
For batch mode, I've added an early checker (line 2581), nearby `needs` checker. 

About cli command - I think it can be `neuro-flow bake --dry-run <batch-name>`.

Reply:
Ok